cmd/webhook/main.go

```diff
@@ -320,12 +321,12 @@ func NewConversionController(ctx context.Context, cmw configmap.Watcher) *contro
 					sourcesv1beta1_:  &sourcesv1beta1.ApiServerSource{},
 				},
 			},
-			sourcesv1alpha2.Kind("PingSource"): {
+			sourcesv1beta1.Kind("PingSource"): {
 				DefinitionName: sources.PingSourceResource.String(),
-				HubVersion:     sourcesv1alpha1_,
+				HubVersion:     sourcesv1alpha2_,
 				Zygotes: map[string]conversion.ConvertibleObject{
-					sourcesv1alpha1_: &sourcesv1alpha1.PingSource{},
```

Comment:
shouldn't this still be here?
do we have three versions of PingSource, right?

Reply:
if so, I think we need the `apis.ConvertToViaProxy(ctx, source, &v1alpha2.PingSource{}, sink)` in the v1alpha1 converters default case (similar thing in the ConvertFrom)

Reply:
do we need it if we remove v1lalpha1?

Reply:
I added it in v1alpha2

Reply:
SG, thanks!